specification.yml

```
@@ -0,0 +1,174 @@
+asyncapi: 3.0.0
+info:
+  title: dp-search-data-importer
+  version: 1.1.0
+  description: |-
+    The *dp-search-data-importer* processes search-data-import events and then 
+    upserts the ElasticSearch index with them.
```

Might be useful to clarify which index it's upserted into…
```suggestion
    upserts the ElasticSearch index specified in the `search-index` field with them.
```
(or similar)

---

I hadn't gone through the code to see if that Search Index field was always used... :D I will now and make it required if it is.

---

If it isn't, it should be. This might be important in future, especially for reindexes.